src/components/ModalConfirmation/styles.module.scss

```diff
@@ -21,3 +21,11 @@ form img {
     overflow-y: hidden;
   }
 }
+
+@media only screen and (max-width: 450px) {
+  .container {
+    width: 100%;
+    height: 100%;
+    overflow-x: hidden;
+  }
+}
```

Comment:
You should run `yarn format` before you push to format everything. You should also run eslint too